From: Mathias Gibbens <gibmat@debian.org>
Description: For the time being, skip this test that hasn't been touched upstream in over five years and is causing an error during build
Forwarded: https://github.com/tinylib/msgp/issues/317
diff --git a/issue185_test.go b/issue185_test.go
index b866108..52363a9 100644
--- a/issue185_test.go
+++ b/issue185_test.go
@@ -1,3 +1,5 @@
+//go:build exclude
+
 package main
 
 import (
